src/Core/Authoring/PackageBuilder.cs

```diff
@@ -455,7 +455,7 @@ private void ReadManifest(Stream stream, string basePath, IPropertyProvider prop
             {
                 if (manifest.Files == null)
                 {
-                    AddFiles(basePath, @"**\*", null);
+                    AddFiles(basePath, string.Format("**{0}*",Path.DirectorySeparatorChar), null);

```

Comment:
Would be good if this would go down a few levels of subdirectories as well

Reply:
Or like all the way down subdirectories - I have not seen it get to a subdirectory under `tools` and pack that in. Which is unfortunate

Reply:
The change looks good tho!

Reply:
I mean the pattern looks lime it should work, but I've not seen it work properly in practice.

Repro: 
1. create a nuspec manifest without a files section. 
1. create a tools folder
2. In the tools folder, create a web folder
3. In the web folder, put a couple of files.
4. Put some files in the tools folder
5. Put some files in the top level folder.
6. run choco pack
7. Rename the nupkg to a zip
8. Unzip the nupkg and inspect what is in there.

Reply:
I'll take a look, and see if I can figure that out that issue as well. That is issue #20, correct?

On a tangent, I actually have 7zip set as the default file association for `.nupkg`s, so I can double-click the compiled package and explore the contents really quickly.

Reply:
@TheCakeIsNaOH where you able to further test the scenario that was described above?

Reply:
@gep13 I did test it, but was unable to replicate the issue..

Reply:
Okay, thanks for confirming.  I am going to move forward with pulling this in.